src/Engine.sol

```diff
@@ -399,25 +467,35 @@ contract Engine is
     /// @inheritdoc IEngine
     function creditAccount(uint128 _accountId, uint256 _amount)
         external
+        payable
         override
     {
-        // ensure account exists (i.e. owner is not the zero address)
-        /// @notice this does not check if the caller is the account owner
-        if (PERPS_MARKET_PROXY.getAccountOwner(_accountId) == address(0)) {
-            revert AccountDoesNotExist();
-        }
-
         credit[_accountId] += _amount;
 
-        /// @dev sUSD transfers that fail will revert
-        SUSD.transferFrom(msg.sender, address(this), _amount);
+        /// @dev $sUSD transfers that fail will revert
+        _SUSD.transferFrom(msg.sender, address(this), _amount);
 
         emit Credited(_accountId, _amount);
     }
 
+    /// @inheritdoc IEngine
+    function creditAccountZap(uint128 _accountId, uint256 _amount)
+        external
+        payable
+        override
+    {
+        // zap $USDC -> $sUSD
+        uint256 usdcAmount = _zapIn(_amount);
```

Comment:
**Question:** Shouldn't the variable here be `sUSDAmount`?

Reply:
yes it should. nice catch